maintainers/maintainer-list.nix

```diff
@@ -4285,6 +4285,12 @@
     githubId = 6074754;
     name = "Hlodver Sigurdsson";
   };
+  hqurve = {
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.